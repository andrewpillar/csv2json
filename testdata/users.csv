id,name,verified,created_at
1,Gordon Freeman,true,19/11/1998
2,Wallace Breen,true,16/11/2004
3,G-Man,false,19/11/1998
4,Barney Calhoun,true,19/11/1998
5,Eli Vance,true,19/11/1998
