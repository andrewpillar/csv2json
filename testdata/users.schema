id          int
verified    bool
created_at  time  02/01/2006  2006-01-02T15:04:05Z
