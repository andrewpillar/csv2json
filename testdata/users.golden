{"created_at":"1998-11-19T00:00:00Z","name":"Gordon Freeman","id":1,"verified":true}
{"created_at":"2004-11-16T00:00:00Z","name":"Wallace Breen","id":2,"verified":true}
{"created_at":"1998-11-19T00:00:00Z","name":"G-Man","id":3,"verified":false}
{"created_at":"1998-11-19T00:00:00Z","name":"Barney Calhoun","id":4,"verified":true}
{"created_at":"1998-11-19T00:00:00Z","name":"Eli Vance","id":5,"verified":true}
